src/app/settings/settings.component.css

```diff
@@ -129,3 +129,13 @@
     background-color: #797979;
     box-shadow: inset -7px -3px 0 0 #ffffff;
 }
+.dark-mode-informative-text{
+    padding-top: 0.3rem;
+    font-size: 0.8rem;
+    color: rgb(180, 180, 180);
```

Comment:
Perhaps use hex colors? Think we use that basically everywhere else. Same for line 140 here.